CoreUtilities/src/au/gov/asd/tac/constellation/utilities/color/ConstellationColor.java

```diff
@@ -424,6 +432,37 @@ public static ConstellationColor fromRgbWithCommaColor(final String color) {
         final float alpha = fields[3] != null ? Float.parseFloat(fields[3]) : 1f;
         return new ConstellationColor(null, red, green, blue, alpha);
     }
+    
+    /**
+     * A fast split method (slightly faster than String.split()).
+     *
+     * The maximum size of the resulting String[] must be known beforehand. The
+     * actual size may be less.
+     *
+     * @param s The string to split.
+     * @param size The expected size of the resulting array.
+     * @param separator The separator character to split on.
+     *
+     * @return A String[] containing the fields that have been split out of the
+     * string.
+     */
+    private static String[] split(final String s, final int size, final char separator) {
```

Comment:
Is this not better suited to the `StringUtilities` class?
Noted that it was always here.

Reply:
The answer is probably (though I think that can be saved for another time 😄)